### DISCUSSION TITLE
Why is The Buypass not working?

### DISCUSSION THREAD

**Discussion by S-PScripts** [3kh0/ext-remover#811] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
The black box appears saying Processing... but then disappears. Is this my extension causing this?

**Comment by AshtonDavies** [comment]
Network or extension often blocks bypassi.com

**Reply by S-PScripts** [reply]
https://www.bypassi.com/ is not banned

**Reply by AshtonDavies** [reply]
Yes I know, maybe not the extension either. I notice that occasionally, and it changes from time to time. That happens when it cannot read the files or the URL doesn't match the json rules.

**Reply by AshtonDavies** [reply]
Like I said, it changes from time to time.

**Comment by S-PScripts** [comment, marked as answer]
wesgsgetg
